.travis.yml

```diff
@@ -19,8 +19,9 @@ install:
   - pip install coverage
 
 script:
-  - flake8 anybox 
-  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox 
+  - flake8 anybox
+  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox
```

Comment:
Just wondering why we can't use the more pythonic:

* `python setup.py flake8`
* `python setup.py tests`